digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala

```diff
@@ -38,29 +57,66 @@ class SearchApi(roadAddressService: RoadAddressService) extends  ScalatraServlet
     }
   }
 
-  get("/road_numbers?") {
+  private val getRoadNumbers: SwaggerSupportSyntax.OperationBuilder =
+    (apiOperation[Seq[Long]]("getRoadNumbers")
+      tags "SearchAPI"
+      summary "getRoadNumbers."
+      )
+
+  get("/road_numbers?", operation(getRoadNumbers)) {
     time(logger, "GET request for /road_numbers?") {
       roadAddressService.getRoadNumbers
     }
   }
 
-  get("/road_address/:road/?") {
+  private val getRoadAddressWithRoadNumber: SwaggerSupportSyntax.OperationBuilder = (
+    apiOperation[List[Map[String, Any]]]("getRoadAddressWithRoadNumber")
+      .parameters(
+        pathParam[Long]("road").description("roadNumber")
+      )
+      tags "SearchAPI"
+      summary "getRoadAddressWithRoadNumber"
+    )
+
+  get("/road_address/:road/?", operation(getRoadAddressWithRoadNumber)) {
     val roadNumber = params("road").toLong
     time(logger, s"GET request for /road_address/$roadNumber/?") {
       val trackCodes = multiParams.getOrElse("tracks", Seq()).map(_.toInt)
       roadAddressService.getRoadAddressWithRoadNumber(roadNumber, Track.applyAll(trackCodes)).map(roadAddressMapper)
     }
   }
 
-  get("/road_address/:road/:roadPart/?") {
+  private val getRoadAddressesFiltered: SwaggerSupportSyntax.OperationBuilder = (
+    apiOperation[List[Map[String, Any]]]("getRoadAddressesFiltered")
+      .parameters(
+        pathParam[Long]("road").description("roadNumber"),
```

Comment:
These descriptions could also be more human friendly

Reply:
fixed